lib/app/services/interceptors/auth_interceptor.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:async';
```

Comment:
Mostly copied from gruene_api_core, removed the constructor logic and the possibility to pass a custom auth repo